src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/Components/FareProducts/FareProductTile.tsx

```diff
@@ -27,13 +47,34 @@ export const FareProductTile = ({
   const styles = useStyles();
   const {t} = useTranslation();
   const {themeName} = useTheme();
+
+  const transportModes = config.transportModes;
+
   const color: StaticColor = accented ? 'background_accent_3' : 'background_0';
   const themeColor = getStaticColor(themeName, color);
+
+  const transportColor = getTransportColorFromModes(
+    transportModes as TransportModesType[],
+  );
+  const transportThemePrimaryColor = getTransportationColor(
+    themeName,
+    transportColor,
+    'primary',
+  );
+  const transportThemeSecondaryColor = getTransportationColor(
+    themeName,
+    transportColor,
+    'secondary',
+  );
+
   const title = useTextForLanguage(config.name);
   const description = useTextForLanguage(config.description);
-  const transportModesText = config.transportModes
-    .map((tm) => t(FareContractTexts.transportMode(tm.mode)))
-    .join('/');
+
+  const transportModesText: string = getFareProductTravelModesText(
```

Comment:
Personally I think explicit typings like this is superfluous. The type is inferred and a good IDE will present this automatically.